Bottom: 55f2adaa06606edf0e093a81c5aeb26835128090
Top:    344dee73de984abcb6b69f444379940569a5ec61
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 16:25:38 +0100

Just make a black background.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 68e7504..e2b4157 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1773,9 +1773,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glGetFloatv (GL_MODELVIEW_MATRIX, (GLfloat *)last_modelview_matrix);
 
   glEnable (GL_STENCIL_TEST);
-  glClearColor (port->offlimits_color.red / 65535.,
-                port->offlimits_color.green / 65535.,
-                port->offlimits_color.blue / 65535.,
+  glClearColor (port->bg_color.red / 65535.,
+                port->bg_color.green / 65535.,
+                port->bg_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -1860,6 +1860,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* Drawing operations as masked to areas where the stencil buffer is '0' */
 //  glStencilFunc (GL_GREATER, 1, 1);             // Draw only where stencil buffer is 0
 
+#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -1890,6 +1891,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
+#endif
 
   // hid_expose_callback (&ghid_hid, &region, 0);
   ghid_draw_everything (&region);
